File: app/src/main/java/yangFenTuoZi/runner/plus/ui/fragment/terminal/TerminalFragment.java
package yangFenTuoZi.runner.plus.ui.fragment.terminal;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;

import yangFenTuoZi.runner.plus.base.BaseFragment;
import yangFenTuoZi.runner.plus.databinding.FragmentTerminalBinding;

public class TerminalFragment extends BaseFragment {

    private FragmentTerminalBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        binding = FragmentTerminalBinding.inflate(inflater, container, false);
        View root = binding.getRoot();
        return root;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }


    @Override
    public void onStart() {
        super.onStart();
    }
}

File: app/src/main/java/yangfentuozi/runner/ui/fragment/terminal/TerminalFragment.kt
package yangfentuozi.runner.ui.fragment.terminal

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import yangfentuozi.runner.base.BaseFragment
import yangfentuozi.runner.databinding.FragmentTerminalBinding

class TerminalFragment : BaseFragment() {
    private var binding: FragmentTerminalBinding? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        binding = FragmentTerminalBinding.inflate(inflater, container, false)
        val root: View? = binding!!.getRoot()
        return root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        binding = null
    }


    override fun onStart() {
        super.onStart()
    }
}